tests/admin/test_admin_users_endpoints.py

```diff
@@ -251,6 +251,33 @@ def test_get_user_noauth(client, db_session):
     r = client.get("/admin/user")
     assert r.status_code == 401
 
+# GET /list_policies test
+
+def test_list_policies(client, admin_user, encoded_admin_jwt db_session):
+    res = client.post(
```

Comment:
Did you test this locally? This should fail because arborist is not running, you should create mock data for the endpoint. For instance https://github.com/chicagopcdc/fence/blob/704125434aaf223139bd7ca85508448c9e808343/tests/conftest.py#L378